packages/document/main-doc/docs/zh/guides/advanced-features/bff/upload.mdx

```diff
@@ -0,0 +1,99 @@
+# 文件上传
+
+BFF 搭配运行时框架提供了文件上传能力，支持一体化调用及纯函数手动调用。
+
+### BFF 函数
+
+首先创建 `api/lambda/upload.ts` 文件：
+
+```ts title="api/lambda/upload.ts"
+export const post = async (req: {formData: Record<string, any>}) => {
```

Comment:
This is not a req(IncomingMessage or Request), and naming it req could lead to a misunderstanding